server/api/slack.js

```diff
@@ -267,15 +267,20 @@ const sendMessageWithAttachments = (botAccessToken, channelId, attachments) => {
   slackBot.chat.postMessage(channelId, "", { attachments });
 };
 
-const sendMessageWithBlocks = (botAccessToken, channelId, blocks) => {
+const sendMessageWithBlocks = async (botAccessToken, channelId, blocks) => {
   const slackBot = new SlackWebClient(botAccessToken);
-  slackBot.chat.postMessage({
+  await slackBot.chat.postMessage({
     channel: channelId,
     text: "",
     blocks
   });
 }
```

Comment:
Why did you need to make this async? You don't seem to be waiting on it.

Reply:
I think my intention was to wait on `handleAuthorizationRequest` and `handleContributionRequest`, so that errors would be caught properly. I must've forgot to propagate the `await`s all the way up.

https://github.com/datadotworld/slack-app/blob/9bf040c4af585fd598777ee95dbdbaff8f50004e/server/controllers/webhook.js#L799-L813